frontend/catalyst/api_extensions/control_flow.py

```diff
@@ -979,6 +986,7 @@ def trace_quantum(self, ctx, device, trace, qrp) -> QRegPromise:
 
                 arg_expanded_classical_tracers = []
                 res_expanded_tracers, out_type = expand_results(
+                    [],
```

Comment:
Will there never be any constants here?

Reply:
Setting correct constants and inputs matters for getting the correct `out_type` (the second result), but we are only interested in the expanded tracers here.